src/components/pages/roster-page/roster-page.tsx

```diff
@@ -319,6 +323,13 @@ export const RosterPage = () => {
     });
   };
 
+  const deleteAllClicked = async () => {
+    // eslint-disable-next-line no-restricted-globals
+    if (confirm('This will permanently delete ALL roster entries?')) {
```

Comment:
Is this just a temporary solution? If so, you can ignore this comment. Otherwise, I'd say we should use a proper confirm dialog for this.

Reply:
Temporary, given the work I was doing on the alert dialog I figured this would get refactored as part of that.